optuna/multi_objective/_hypervolume/wfg.py

```diff
@@ -48,11 +50,13 @@ def _compute_rec(self, solution_set: np.ndarray) -> float:
         return volume
 
     def _compute_exclusive_hv(self, point: np.ndarray, solution_set: np.ndarray) -> float:
-        volume = _compute_2points_volume(point, self._reference_point)
+        volume = _compute_2points_volume(point, np.array(self._reference_point))
```

Comment:
Let's avoid copies. Or better, you could add an assertion that it's not `None` prior to this line similar to `_compute_rec`.
```suggestion
        volume = _compute_2points_volume(point, np.asarray(self._reference_point))
```